### PR TITLE
Added Czech translation

### PR DESCRIPTION
**Pull request by pavreh**

### CODE REVIEW

File: youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po [hunk 1]
```diff
@@ -0,0 +1,589 @@
...
+msgid "Warning"
+msgstr "Upozornění"
+
+#: youtube_dl_gui/mainframe.py:105
+msgid "Add"
+msgstr "Přidat"
+
+#: youtube_dl_gui/mainframe.py:106
+msgid "Download list"
+msgstr "Seznam stahovaných"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po]: This *msgstr* translates to "List downloaded". Shouldn't this be "Seznam stahování" or am i wrong?

File: youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po [hunk 2]
```diff
@@ -0,0 +1,589 @@
...
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "Stahuje se nejnovější youtube-dl. Počkejte prosím..."
+
+#: youtube_dl_gui/mainframe.py:131
+msgid "Youtube-dl download failed [{0}]"
+msgstr "Stahování youtube-dl selhalo [{0}]"
+
+#: youtube_dl_gui/mainframe.py:132
+msgid "Successfully downloaded youtube-dl"
+msgstr "youtube-dl byl úspěšně stažen"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po]: You should change *youtube-dl* to ***Y**outube-dl*.

File: youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po [hunk 3]
```diff
@@ -0,0 +1,589 @@
...
+msgid "Ignore youtube-dl config"
+msgstr "Ignorovat nastavení youtube-dl"
+
+#: youtube_dl_gui/optionsframe.py:871
+msgid "No mtime"
+msgstr ""
+
+#: youtube_dl_gui/optionsframe.py:872
+msgid "Prefer native HLS"
+msgstr ""
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po]: Why not translate this? (https://translate.google.gr/#en/cs/Prefer)

File: youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po (hunk 1 above)
Comment (pavreh) [reply youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po]: "Seznam stahovaných" means "List of some things, which are downloading". But you are right, "Seznam stahování" is common in a software.

File: youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po (hunk 2 above)
Comment (pavreh) [reply youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po]: ok

File: youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po (hunk 3 above)
Comment (pavreh) [reply youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po]: Translated

**Comment by MrS0m30n3** [comment]
Thank you very much for your time!
I will merge this as soon as i'm done with the other translations. :+1:

**Pull request merged**